### PR TITLE
VirusTotal Fix File Sandbox Report Error Handling

### PR DESCRIPTION
**Pull request by mohlcyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Fixed an issue where the **vt-file-sandbox-report** command was returning error if hash was not found.
This generated errors in Playbook File Enrichment - Virus Total (API v3)

## Description
Improved error handling. No Errors will be raised if hash is unknown to Virustotal anymore. Updated error handling in Playbook.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/VirusTotal/ReleaseNotes/2_6_18.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#### Integrations
+##### VirusTotal (API v3)
+- Fixed an issue where the **vt-file-sandbox-report** command was returning error if hash was not found.
+- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
+
+#### Playbooks
+##### File Enrichment - Virus Total (API v3)
+- Fixed error handling for Enriching a behavioral relationship task.
```
Comment (jbabazadeh) [inline Packs/VirusTotal/ReleaseNotes/2_6_18.md:8]: Added an error handling for Enriching a behavioral relationship task.

**Review by jbabazadeh — COMMENTED** [review]
Hi @mohlcyber 
Thank you for your contribution!Good work :)
Please see my comments

**Review by melamedbn — APPROVED** [review]

File: Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py [hunk 2]
```diff
@@ -2297,26 +2298,31 @@ def file_sandbox_report_command(client: Client, args: dict) -> CommandResults:
...
+                removeNull=True,
+                headerTransform=underscoreToCamelCase
+            ),
+            outputs=data,
+            raw_response=raw_response
+        )
+    elif raw_response.get('error', {}).get('code') == 'NotFoundError':
+        return CommandResults(readable_output=f'{file_hash} not found.')
+    else:
+        raise DemistoException("Error API Quota Exceeded.")
```
Comment (danipv) [inline Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py:2325]: Hello Martin, thanks a lot for your contribution. Have you consider to return a `CommandResults` when quota exceeded error? that is the usual behavior in the rest of the commands like file_command: https://github.com/VirusTotal/content/blob/master/Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py#L1811

Comment (mohlcyber) [reply Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py]: Hi @danipv,
thanks for your feedback. Great point. I adjusted the returned CommandResults to align to the usual behavior in the integration code. Please review the last commit.

Comment (mohlcyber) [reply Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py]: @danipv i also leveraged the same build_quota_exceeded_file_output and build_unknown_file_output functions as i think the same output applies to the sandbox reports as well.

Comment (danipv) [reply Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py]: Love that. Thanks a lot. @jbabazadeh LGTM.

**Review by danipv — APPROVED** [review]

**Comment by mohlcyber** [comment]
Demo has been recorded: https://drive.google.com/file/d/106qxSGMvkiGW2or8wS28vYW9PCSoTNXd/view?usp=sharing

**Comment by jbabazadeh** [comment]
Hi @mohlcyber 
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Comment by mohlcyber** [comment]
Awesome thanks so much @jbabazadeh for your support. Much appreciated.

**Pull request merged**